diff --git a/ServerLibrary/Models/PlayerObject.cs b/ServerLibrary/Models/PlayerObject.cs
--- a/ServerLibrary/Models/PlayerObject.cs
+++ b/ServerLibrary/Models/PlayerObject.cs
@@ -9000,11 +9000,6 @@ namespace Server.Models
 
                 buff = Buffs.FirstOrDefault(x => x.Type == BuffType.MagicShield);
 
-                if (buff != null)
-                {
-                    buff.RemainingTime -= TimeSpan.FromMilliseconds(power * 25);
-                    Enqueue(new S.BuffTime { Index = buff.Index, Time = buff.RemainingTime });
-                }
-
                 power -= power * Stats[Stat.MagicShield] / 100;
             }
