diff --git a/LibraryCore/SystemModels/MapInfo.cs b/LibraryCore/SystemModels/MapInfo.cs
--- a/LibraryCore/SystemModels/MapInfo.cs
+++ b/LibraryCore/SystemModels/MapInfo.cs
@@ -130,6 +130,21 @@ namespace Library.SystemModels
         }
         private int _FireWallCount;
 
+        public bool NoPets
+        {
+            get { return _NoPets; }
+            set
+            {
+                if (_NoPets == value) return;
+
+                var oldValue = _NoPets;
+                _NoPets = value;
+
+                OnChanged(oldValue, value, "NoPets");
+            }
+        }
+        private bool _NoPets;
+
         public bool CanHorse
         {
             get { return _CanHorse; }
diff --git a/ServerLibrary/Envir/Config.cs b/ServerLibrary/Envir/Config.cs
--- a/ServerLibrary/Envir/Config.cs
+++ b/ServerLibrary/Envir/Config.cs
@@ -137,4 +137,5 @@ namespace Server.Envir
         public static TimeSpan DeadDuration { get; set; } = TimeSpan.FromMinutes(1);
         public static TimeSpan HarvestDuration { get; set; } = TimeSpan.FromMinutes(5);
+        public static int MaxBossTames { get; set; } = 1;
         public static int MysteryShipRegionIndex { get; set; } = 0;
         public static int LairRegionIndex { get; set; } = 0;
