diff --git a/LibraryCore/Enum.cs b/LibraryCore/Enum.cs
--- a/LibraryCore/Enum.cs
+++ b/LibraryCore/Enum.cs
@@ -15,6 +15,8 @@
         Wizard = 1,
         Taoist = 2,
         Assassin = 3,
+        Archer = 4,
+        Monk = 5,
     }
 
     public enum AttackMode : byte
@@ -74,12 +76,14 @@
         Wizard = 2,
         Taoist = 4,
         Assassin = 8,
+        Archer = 16,
+        Monk = 32,
         [Description("Warrior, Wizard, Taoist")]
         WarWizTao = Warrior | Wizard | Taoist,
         [Description("Wizard, Taoist")]
         WizTao = Wizard | Taoist,
         [Description("Warrior, Assassin")]
         AssWar = Warrior | Assassin,
-        All = WarWizTao | Assassin
+        All = WarWizTao | Assassin | Archer | Monk
     }
 }
